doc/src/cylc-user-guide/suiterc.tex

```diff
@@ -754,16 +754,99 @@ \subsection{[scheduling]}
+
+This can be used to set the initial cycle point time relative to the current time.
+
+Two additional commands, \lstinline=next= and \lstinline=previous=, can be used when setting the initial cycle point.
+
+The syntax uses truncated ISO8601 time representations, and is of the style: \lstinline=next(Thh:mmZ)=, \lstinline=previous(T-mm)=; e.g.
+\begin{myitemize}
+    \item {\lstinline.initial cycle point = next(T15:00Z).}
+    \item {\lstinline.initial cycle point = previous(T09:00).}
+    \item {\lstinline.initial cycle point = next(T12).}
+    \item {\lstinline.initial cycle point = previous(T-20).}
+\end{myitemize}
+Examples of interpretation are given in figure \ref{relative initial cycle point time syntax interpretation}.
+
+A list of times, separated by semicolons, can be provided, e.g. \lstinline=next(T-00;T-15;T-30;T-45)=.
+At least one time is required within the brackets, and if more than one is given, the major time unit in each (hours or minutes) should all be of the same type.
+
+If an offset from the specified date or time is required, this should be used in the form: \lstinline=previous(Thh:mm)= \begin{math} \pm \end{math}\lstinline=PxTy=
+in the same way as is used for determining cycle periods, e.g.
+\begin{myitemize}
+    \item {\lstinline.initial cycle point = previous(T06) +P1D.}
+    \item {\lstinline.initial cycle point = next(T-30) -PT1H.}
+\end{myitemize}
+The section in the bracket attached to the next/previous command is interpreted first, and then the offset is applied.
+
+The offset can also be used independently without a \lstinline=next= or \lstinline=previous= command, and will be interpreted as an offset from ``now''.
+
+\begin{figure}[H]
+\centering
+\caption{Examples of setting relative initial cycle point for times and offsets \\
+using now = 2018-03-14T15:12Z (and UTC mode)}
+\begin{tabular}{ | l | l | }
+\hline
+Syntax & Interpretation \\
+\hline
+\lstinline=next(T-00)= & 2018-03-14T16:00Z \\
+\lstinline=previous(T-00)= & 2018-03-14T15:00Z \\
+\lstinline=next(T-00; T-15; T-30; T-45)= & 2018-03-14T15:15Z \\
+\lstinline=previous(T-00; T-15; T-30; T-45)= & 2018-03-14T15:00Z \\
+\lstinline=next(T00)= & 2018-03-15T00:00Z \\
+\lstinline=previous(T00)= & 2018-03-14T00:00Z \\
+\lstinline=next(T06:30Z)= & 2018-03-15T06:30Z \\
+\lstinline=previous(T06:30) -P1D= & 2018-03-13T06:30Z \\
+\lstinline=next(T00; T06; T12; T18) = & 2018-03-14T18:00Z \\
+\lstinline=previous(T00; T06; T12; T18) = & 2018-03-14T12:00Z \\
+\lstinline=next(T00; T06; T12; T18) +P1W = & 2018-03-21T18:00Z \\
+\lstinline=PT1H= & 2018-03-14T16:12Z \\
+\lstinline=-P1M= & 2018-02-14T15:12Z \\
+\hline
+\end{tabular}
+\label{relative initial cycle point time syntax interpretation}
+\end{figure}
+
+The relative intial cycle point also works with truncated dates, including weeks and ordinal date, using ISO8601 truncated date representations.
```

Comment:
"with truncated dates, including weeks and ordinal date" does the last `date` need to be in plural? Probably not??...

Reply:
I don't think the plurals need to agree there, but I am not a grammar expert!

Reply:
Ditto for me! And not even a native speaker 😬 Let's mark as resolved.